mne/channels/montage.py

```diff
@@ -313,7 +313,7 @@ def read_montage(kind, ch_names=None, path=None, unit='m', transform=False):
         ch_names_ = data['name'].astype(str).tolist()
         pos = np.vstack((data['x'], data['y'], data['z'])).T
     elif ext in ('.loc', '.locs', '.eloc'):
-        ch_names_ = np.loadtxt(fname, dtype='S4',
+        ch_names_ = np.loadtxt(fname, dtype='S8',
```

Comment:
can we just let numpy find out the size it needs? maybe 8 is still not enough for some.